src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -148,12 +151,14 @@ public function register(Container $app)
         };
 
         // generate the build-in authentication factories
-        foreach (array('logout', 'pre_auth', 'form', 'http', 'remember_me', 'anonymous') as $type) {
+        foreach (array('logout', 'pre_auth', 'guard', 'form', 'http', 'remember_me', 'anonymous') as $type) {
             $entryPoint = null;
             if ('http' === $type) {
                 $entryPoint = 'http';
             } elseif ('form' === $type) {
                 $entryPoint = 'form';
+            } elseif ('guard' === $type) {
+                $entryPoint = 'guard';
             }
```

Comment:
:+1: this is consistent with the full-stack: we don't replace the entry point of other auth methods, just to be safe